driver/razeraccessory_driver.c

```diff
@@ -884,10 +880,10 @@ static ssize_t razer_attr_write_matrix_custom_frame(struct device *dev, struct d
             return -EINVAL;
         }
 
-        row_id = buf[offset++];
-        start_col = buf[offset++];
-        stop_col = buf[offset++];
-        row_length = ((stop_col+1) - start_col) * 3;
+        unsigned char row_id = buf[offset++];
+        unsigned char start_col = buf[offset++];
+        unsigned char stop_col = buf[offset++];
+        unsigned char row_length = ((stop_col+1) - start_col) * 3;
```

Comment:
Generally variables in Linux kernel are declared on top, but from what I find this seems to be allowed now with C11 kernel target https://lwn.net/Articles/885941/ , at least in loops

Reply:
After thinking about it, I don't think moving the variables down provides anything useful, and I think it actually decreases readability. I'll revert this change.